source/about/licensing-and-subscription.rst

```diff
@@ -226,32 +82,30 @@ The affected Enterprise features include, but are not limited to, the following:
     
     Mentions for AD/LDAP groups are not shown in the autocomplete menu.
     
-    Group mentions are no longer highlighted in text and do not trigger new notifications.", "Use the `CLI <https://docs.mattermost.com/manage/command-line-tools.html#mattermost-group>`__ or the `mmctl <https://docs.mattermost.com/manage/mmctl-command-line-tool.html#mmctl-group>`__ to modify group sync settings for the team/channel."
+    Group mentions are no longer highlighted in text and do not trigger new notifications.", "Use the `CLI <https://docs.mattermost.com/manage/command-line-tools.html#mattermost-group>`_ to modify group sync settings for the team/channel."
```

Comment:
We'll want to include the mmctl equivalent here as well

Reply:
Applied in new file created when I changed the file name.